format/tmux/formater.go

```diff
@@ -164,7 +179,7 @@ func (f *Formater) divergence() {
 
 func (f *Formater) flags() {
 	f.clear()
-	f.b.WriteString(" - ")
+	f.b.WriteString(f.Symbols.Delimiter)
```

Comment:
We can't just write the delimiter before the flags, since the flags could be the first component to be shown if the user has placed the `flags` component first in their configuration file. It will need to be intertwined in-between both components.